scripts/release.py

```diff
@@ -359,6 +361,25 @@ def prs_by_email(start_ref, end_ref):
     The dictionary is alphabetically ordered by email address
     The pull request list is ordered by merge date
     """
+    # `emails` maps from other_emails to primary email, based on people.yaml.
+    emails = {}
+    try:
+        people_resp = requests.get(PEOPLE_YAML)
+        if people_resp.status_code != 200:
+            raise Exception("status {}: {}".format(people_resp.status_code, people_resp.text))
+        people = yaml.safe_load(people_resp.text)
+    except Exception as e:
+        # Hmm, muddle through without canonicalized emails...
+        message = (
+            "Warning: could not fetch people.yaml: {message}".format(message=e.message)
+        )
+        print(colorize("red", message), file=sys.stderr)
+    else:
+        for person in people.itervalues():
+            if 'other_emails' in person:
```

Comment:
Yes, the point is to find merge emails which are listed in other_emails, so that we can instead use the email field.  This map only need to map from other_email to email.  Once this script is updated, we can add more other_emails into people.yaml.